student/intro.php

```diff
@@ -81,8 +80,20 @@ function student_intro() {
 		// Ima li termina na koje se može prijaviti?
 		$q17 = db_query("select count(*) from ispit_termin where ispit=$r15[0] and datumvrijeme>=NOW()");
 		if (db_result($q17,0,0)>0) {
-			$code_poruke["i".$r15[0]] = "<b>$r15[4]:</b> Objavljeni termini za ispit $r15[2]. <a href=\"?sta=student/prijava_ispita&predmet=$r15[8]&ag=$r15[9]\">Prijavite se!</a><br /><br />\n";
-			$vrijeme_poruke["i".$r15[0]] = $r15[3];
+            // Provjeravamo da li je student apsolvent i da li je ispit naznačen kao ispit za apsolventa
+            $s_s = (int)db_fetch_row(db_query("select status from student_studij where student = $userid"))[0];

```

Comment:
Ovdje bi trebalo koristiti db_get jer ti radiš cast array u int što je malo čudno.